browser/components/NewStudentForm.js

```diff
@@ -1,61 +1,75 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 
 export default class NewStudentForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
       firstName: '',
       lastName: '',
-      email: '',
+      email: ''
     };
-    this.handleChange = this.handleChange.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
   }
 
-  handleChange(event) {
+  handleChange = (e) => {
```

Comment:
I don't think the arrow syntax is standard yet in our curriculum (ie: not all instructors/workshops cover it) and unsure if all fellows are aware of this new syntax. Might be safer (for now) to keep the non-arrow syntax + binding to be consistent with the standard lecture + workshops until we change the curriculum to match.

Reply:
I'm still going to approve this PR since the code the student writes will work either way. If this arrow syntax is the code we choose to merge, maybe a comment noting the difference + why we no longer need bind could be helpful.